core/client/app/templates/settings/tags/settings-menu.hbs

```diff
@@ -53,17 +54,19 @@
 
                 <div class="settings-menu-content">
                     <form>
-                    <div class="form-group">
+                    {{#gh-form-group errors=activeTag.errors property="meta_titel"}}
```

Comment:
Argh! I was waiting for this to go green and then I gave it one last look and spotted this... sorry!